### PR TITLE
update keys HRP - fund6

### PR DESCRIPTION
**Pull request by Zeegomo**

### CODE REVIEW

**Review by dkijania — APPROVED** [review]

**Review by danielSanchezQ — APPROVED** [review]

**Comment by mzabaluev** [comment]
Hardcoding the curve in the HRP stopped making sense when the backend became selectable.
Can we still extract the curve ID from the backend and use that in the HRP for debug purposes, and making it easy to diagnose when a key generated for a different curve is used?

**Comment by Zeegomo** [comment]
Failure is unrelated

File: jcli/src/jcli_lib/certificate/mod.rs [hunk 1]
```diff
@@ -69,8 +69,8 @@ pub enum Error {
     KeyNotFound { index: usize },
     #[error("Invalid input, expected Signed Certificate or just Certificate")]
     ExpectedSignedOrNotCertificate,
-    #[error("Invalid data")]
-    InvalidBech32(#[from] bech32::Error),
+    #[error("Invalid bech32 data")]
```
Comment (mzabaluev) [inline jcli/src/jcli_lib/certificate/mod.rs:72]: Maybe "invalid Bech32 address string" would be more descriptive.

File: jcli/src/jcli_lib/vote/mod.rs [hunk 2]
```diff
@@ -17,8 +16,8 @@ pub enum Error {
     Hex(#[from] hex::FromHexError),
     #[error("base64 decode error")]
     Base64(#[from] base64::DecodeError),
-    #[error("bech32 decode error")]
-    Bech32(#[from] bech32::Error),
+    #[error("bech32 error")]
```
Comment (mzabaluev) [inline jcli/src/jcli_lib/vote/mod.rs:19]: "Bech32 format error"?

**Review by mzabaluev — APPROVED** [review]

File: jcli/src/jcli_lib/certificate/mod.rs (hunk 1 above)
Comment (Zeegomo) [reply jcli/src/jcli_lib/certificate/mod.rs:72]: It's not used only for keys though, certificates are bech32 encoded as well while they are stored locally

**Pull request merged**